src/python/gi/overrides/BlockDev.py

```diff
@@ -191,10 +191,10 @@ def btrfs_check(mountpoint, extra=None, **kwargs):
 __all__.append("btrfs_check")
 
 
-_crypto_luks_format = BlockDev.crypto_luks_format
+_crypto_luks_format = BlockDev.crypto_luks_format_luks2
```

Comment:
I think this deserves a comment that ``crypto_luks_format_luks2(extra=None)`` is the same as ``luks_format()``.